ITI/TF/Volume2/ITI-18.html

```diff
@@ -879,6 +895,10 @@ <h6 id="3.18.4.1.2.3">3.18.4.1.2.3 Query Request Parameters – Coding Style</h6
                     <sup>2</sup>
                     Shall be coded according to specification in <a href="#3.18.4.1.2.3.4">Section 3.18.4.1.2.3.4 Coding of Code/Code-Scheme</a>.
                 </p>
+                <p>
+                    <sup>3</sup>
+                    The use of this optional parameter is only expected if the Document Consumer supports the “Target Communities” Option (See Section 10.2.11 Target Communities Option).
+                </p>
                 <p class="heading8" id="3.18.4.1.2.3.7.3">3.18.4.1.2.3.7.3 FindFolders</p>
```

Comment:
FindFolders adHocQuery is missing the $targetCommunityIdList parameter in the table. line 948